README.md

```diff
@@ -1 +1,117 @@
-# business-summit
\ No newline at end of file
+# business-summit
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Business Summit] <a name="about-project"></a>
+
+Business Summit is the official web-page for 2023 Latin America Business Summit taking place in Brasilia. The logo was designed by Juan Gurdian.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+VS Code editor
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS and JAVASCRIPT
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML file]**
+- **[CSS file]**
+- **[JS file]**
+- **[Linters installed: Lighthouse, Webhint, Stylelint, ESlint and node_modules]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ https://eamador1.github.io/portfolio/
```

Comment:
- [ ] Your Live demo link is not working. Please make sure to fix and also make sure that all pages are working as expected ✔️